benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp

```diff
@@ -687,17 +687,41 @@ void GraphicsBenchmarkApp::DrawExtraInfo()
     GetGraphicsQueue()->GetTimestampFrequency(&frequency);
 
     ImGui::Columns(2);
-    const float gpuWorkDuration = static_cast<float>(mGpuWorkDuration / static_cast<double>(frequency)) * 1000.0f;
+    const float gpuWorkDuration_in_sec = static_cast<float>(mGpuWorkDuration / static_cast<double>(frequency));
+    const float gpuWorkDuration_in_ms  = gpuWorkDuration_in_sec * 1000.0f;
     ImGui::Text("GPU Work Duration");
     ImGui::NextColumn();
-    ImGui::Text("%f ms ", gpuWorkDuration);
+    ImGui::Text("%.2f ms ", gpuWorkDuration_in_ms);
     ImGui::NextColumn();
 
     ImGui::Columns(2);
     const float gpuFPS = static_cast<float>(frequency / static_cast<double>(mGpuWorkDuration));
     ImGui::Text("GPU FPS");
     ImGui::NextColumn();
-    ImGui::Text("%f fps ", gpuFPS);
+    ImGui::Text("%.2f fps ", gpuFPS);
+    ImGui::NextColumn();
+
+    const uint32_t width  = GetSwapchain()->GetWidth();
+    const uint32_t height = GetSwapchain()->GetHeight();
+    ImGui::Columns(2);
+    ImGui::Text("Swapchain resolution");
+    ImGui::NextColumn();
+    ImGui::Text("%d x %d", width, height);
+    ImGui::NextColumn();
+
+    const uint32_t quad_count       = pFullscreenQuadsCount->GetValue();
+    const float    data_write_in_gb = static_cast<float>(width) * static_cast<float>(height) * 4 / (1024.f * 1024.f * 1024.f) * quad_count;
```

Comment:
fixed

Reply:
Actually, i changed it to 
```
(static_cast<float>(width) * static_cast<float>(height) * 4.f * quad_count) / (1024.f * 1024.f * 1024.f);
```
This is because `width * height * 4 * quad_count` could be a huge value (for 4k vr device with a large quad count) that is larger than the uint32 max. (3840x2x3840x4x1000)